vent/sqldb/sqldb.go

```diff
@@ -20,17 +20,21 @@ type SQLDB struct {
 
 // NewSQLDB delegates work to a specific database adapter implementation,
 // opens database connection and create log tables
-func NewSQLDB(dbAdapter, dbURL, schema string, log *logger.Logger) (*SQLDB, error) {
+func NewSQLDB(dbAdapter string, dbURL string, schema string, log *logger.Logger) (*SQLDB, error) {
 	db := &SQLDB{
```

Comment:
` NewSQLDB(dbAdapter string, dbURL string, schema string, log *logger.Logger) (*SQLDB, error)` should be
` NewSQLDB(dbAdapter, dbURL, schema string, log *logger.Logger) (*SQLDB, error)` instead

Reply:
ready